lib/types.ts

```diff
@@ -263,6 +263,11 @@ type Collection<TKey extends CollectionKeyBase, TValue, TMap = never> = {
 /** Represents the base options used in `Onyx.connect()` method. */
 type BaseConnectOptions = {
     initWithStoredValues?: boolean;
+    /**
+     * If true, it will call the callback function even if the value might hasn't changed.
```

Comment:
I think you left an error here, right? "Value might hasn't changed"

Reply:
Hm maybe my wording wasn't precise here.
When this value is `true` it can happen that the Onyx.conenct callback gets called with the same value multiple times (basically bypass the improvement)

Do you have an idea how to rephrase it to make it more obvious / better understandable?

Reply:
Currently discussing if this param is really needed

https://expensify.slack.com/archives/C05LX9D6E07/p1721746491534399?thread_ts=1721742421.298419&cid=C05LX9D6E07